add cpu support for docker, update readme

vikramkalabi opened the pull request:
Updated docker to run with or without GPU

New instructions have been added in the README.

ziv-lin commented:
It seems that the pull request is conflicting with the base branch. Can you resolve the conflicts and resubmit the pull request again?

vikramkalabi commented:
I have added `docker/run_gpu` file for enabling GPU with docker. `docker/run` has  been updated to run without GPU. Added a steps to run docker with GPU. Could you please tell me what conflict are you seeing?

ziv-lin commented:
Sorry for the late reply, I have just merged your pull request.
